x-pack/dockerlogbeat/docs/install.asciidoc

```diff
@@ -0,0 +1,109 @@
+[[log-driver-installation]]
+== Install and configure the {log-driver}
+
+++++
+<titleabbrev>Install and configure</titleabbrev>
+++++
+
+experimental[]
+
+[float]
+=== Before you begin
+
+Make sure your system meets the following prerequisites:
+
+* Docker: {docker-version}
+* {stack}: Version 7.6.0 and later
+
+//REVIEWERS: Are there any other prereqs?
```

Comment:
@fearful-symmetry Are there any other prereqs that I'm missing here?

Reply:
I don't believe so.